pr-check.js

```diff
@@ -171,7 +171,25 @@ function prChecker(config, argLog, argStore, GH, mailer) {
             pr.contribStatus[username] = "undetermined affiliation";
             return "undetermined affiliation";
           }
-          let result = await w3ciprcheck(w3c, user.w3capi, user.displayName, repoGroups, store);
+
+          let groups = [];
+          const types = {
```

Comment:
nit: this could be defined as a module-level object since it is essentially immutable

Reply:
Thanks! PR updated.